src/admin/admin-list/admin-list.component.ts

```diff
@@ -46,6 +47,7 @@ export class AdminListComponent implements OnInit {
       this.adminService.className = params.className;
       this.adminService.loadSchema()
         .then((response) => {
+          this.changeDetectorRef.markForCheck();
```

Comment:
So I'm guessing you're using a tool here that's specifically helpful when you need to trigger a view lifecycle hook at a given time? Is that what's going on?

Reply:
oh no, i was just using that as a sanity check at one point because the page was not actually being updated, i can probably remove this.  actually i will, after i do the test flight for mc machinery.